.github/workflow/md-links.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: md-links
+
+"on":
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  md-links:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@master
+      - name: markdown-link-check
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with:
+          use-verbose-mode: "yes"
+          folder-path: "source"
```

Comment:
```suggestion
        with:
          use-verbose-mode: 'yes'
          folder-path: 'source'
          check-modified-files-only: 'yes'
```

Reply:
> Will this fetch each and every link on each PR?

@balloob The `check-modified-files-only` will make it only check files that were changed.